From: Guillaume Brochu <guillaume.brochu@gmail.com>
Date: Sun, 21 Oct 2018 15:02:09 -0400
Subject: Reproducible-Builds

Do not include compilation date and time in the compiled binary file
in order to get reproducible builds
---
 src/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 03271e1..908f98b 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -61,8 +61,9 @@ compilation.cmx: $(GWD_OBJS1:.cmo=.cmx) $(GWD_OBJS2.cmo=.cmx) gwd.cmx
 compilation.ml: always
 	> compilation.ml
 	echo "open Def;" >> compilation.ml
-	echo "value (wd, d, h) = ($$(date "+%u, {day=%d;month=%m;year=%Y;prec=Sure;delta=0}, \"%T %Z\""));" >> compilation.ml
-	echo "value ct conf = Printf.sprintf \" (%4d-%02d-%02d %s)\" d.year d.month d.day h;" >> compilation.ml
+	#echo "value (wd, d, h) = ($$(date "+%u, {day=%d;month=%m;year=%Y;prec=Sure;delta=0}, \"%T %Z\""));" >> compilation.ml
+	#echo "value ct conf = Printf.sprintf \" (%4d-%02d-%02d %s)\" d.year d.month d.day h;" >> compilation.ml
+	echo "value ct conf = Printf.sprintf \"(compilation time not available, wiki.debian.org/ReproducibleBuilds)\";" >> compilation.ml
 	echo "Util.compilation_time_hook.val := ct;" >> compilation.ml
 
 clean::
